src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -30,6 +31,7 @@ Cypress.Commands.add('fillVaTextarea', (field, value) => {
       .shadow()
       .find('textarea')
       .as('currentElement')
+      .clear({ force: true })
```

Comment:
^ This clear cmd adds resilience to flakiness from fields still being disabled when this method starts.